.github/workflows/_build_docker.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2022 - 2023, Oracle and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl/.
+
+# This is a reuseable workflow to build and test the Macaron Docker image. Note that this workflow does not
+# push the built Docker image. This reuseable workflow needs two mandatory inputs:
+#   artifact-name: The name of the artifact to download. This is the same artifact generated by _build.yaml
+#   artifact-sha256: The hash to verify against the downloaded artifact.
+
+name: Build and push Docker image
+on:
+  workflow_call:
+    inputs:
+      artifact-name:
+        required: true
+        type: string
+      artifact-sha256:
+        required: true
+        type: string
+permissions:
+  contents: read
+
+jobs:
+  build-docker:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: read
+    steps:
+    - name: Check out repository
```

Comment:
Please add a new line bellow `steps` to be consistent with other workflows.

Reply:
Fixed https://github.com/oracle-samples/macaron/pull/76/commits/6d2ab69b14a7377c3735644354b069b9171b9f78